packages/contracts-bedrock/src/L2/CrossL2Inbox.sol

```diff
@@ -56,8 +67,8 @@ contract CrossL2Inbox is ICrossL2Inbox, ISemver, TransientReentrancyAware {
     bytes32 internal constant CHAINID_SLOT = 0x6e0446e8b5098b8c8193f964f1b567ec3a2bdaeba33d36acb85c1f1d3f92d313;
 
     /// @notice Semantic version.
-    /// @custom:semver 1.0.0-beta.3
-    string public constant version = "1.0.0-beta.3";
+    /// @custom:semver 1.1.0-beta.3
+    string public constant version = "1.1.0-beta.3";
```

Comment:
just to double check, is this version bump correct?
should we just bump `.3` to `.4`?

Reply:
Based on their guidelinses, `minor` applies for changes that add functionality but doesn't break anything from the current interface. `Patch` instead is for things like docs
https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/STYLE_GUIDE.md#versioning

Reply:
Perfect then!